git_hooks/post-checkout

```diff
@@ -2,4 +2,4 @@
 #
 # Writes git revision into git_revision file
 #
-git show-ref refs/heads/master | cut -d " " -f 1  > git_revision
+git rev-parse HEAD | cut -d " " -f 1  > git_revision
```

Comment:
Just getting a feel for this PR... I already did that on this repo's checkout with a branch there... but... if someone has `git_revision` already on their machine it's going to present an issue.

btw when does Travis get finished typically?